app/controllers/agenda-items.ts

```diff
@@ -26,6 +29,10 @@ export default class AgendaItemsController extends Controller {
   @tracked loading = false;
   @tracked errorMsg = '';
 
+  setup() {
+    this.agendaItems = [...this.model.agendaItems];
```

Comment:
This can also be done in the `setupController` method of the route. I just did it like this because of personal preference, but I can change it around if needed.

Reply:
```suggestion
    this.agendaItems = [...this.model.agendaItems.toArray()];
```

Related to https://github.com/lblod/frontend-burgernabije-besluitendatabank/pull/46/files#diff-45b09f498e20729ac9a181d2ba5ae273a7466b2be5dfa51c4318155a93e5161eR26 changes. Here no complain because `agendaItems` is typed `any`, but would add a `toArray()` too.